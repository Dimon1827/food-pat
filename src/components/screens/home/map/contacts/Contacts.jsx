function Contacts() {
  return (
    <div className="map__contacts map-contacts">
      <p className="map-contacts__text">
        Наш центр можете отыскать по этому адресу
      </p>
      <div className="map-contacts__info">
        <address className="map-contacts__address">
          улица Багратионосовкая, д. 5
        </address>
        <address className="map-contacts__address">Москва</address>
      </div>
    </div>
  );
}

export default Contacts;
