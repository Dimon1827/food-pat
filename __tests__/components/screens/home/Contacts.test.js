import { render } from '@testing-library/react';
import Contacts from '@/components/screens/home/map/contacts/Contacts';

describe('Contacts', () => {
  it('Contacts renders correctly', () => {
    const { getByTestId } = render(<Contacts />);
    expect(getByTestId('contacts')).toMatchSnapshot();
  });
});