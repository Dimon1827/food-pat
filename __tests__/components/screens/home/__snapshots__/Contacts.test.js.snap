// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contacts Contacts renders correctly 1`] = `
<div
  class="map__contacts map-contacts"
  data-testid="contacts"
>
  <p
    class="map-contacts__text"
  >
    Наш центр можете отыскать по этому адресу
  </p>
  <div
    class="map-contacts__info"
  >
    <address
      class="map-contacts__address"
    >
      улица Багратионосовкая, д. 5
    </address>
    <address
      class="map-contacts__address"
    >
      Москва
    </address>
  </div>
</div>
`;
